controllers/app.controller.js

```diff
@@ -71,3 +72,20 @@ exports.postComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.updateArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (!inc_votes) {
+    return next({
+      status: 400,
+      msg: "Bad Request",
```

Comment:
As this is custom error handling, it should really be rejected in your model, then the next in the catch block should send it back to your app error handling middleware in order to send a custom error 👍